kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -34,10 +53,45 @@ var ContentAreaView = BaseView.extend({
         this.close();
         // set the new view as the current view
         this.currently_shown_view = view;
+
         // show the view
         this.$(".content").html("").append(view.$el);
     },
 
+    show_rating: function() {
+        if( typeof  window.statusModel.get("user_id") === "undefined" ) {
+            return;  // Let this be a no-op if the user isn't logged in.
+        }
+        if( typeof this.rating_view === "undefined" ) {
+            this.rating_view = this.add_subview(RatingView, {
+                el: this.$("#rating-container-wrapper"),
+                model: new RatingModel({
+                    "user": window.statusModel.get("user_uri"),
+                    "content_kind": this.model.get("kind"),
+                    "content_id": this.model.get("id")
+                })
+            });
+        }
+        if( this.rating_view.model.get("content_id") !== this.model.get("id") ) {
+            var self = this;
+            this.content_rating_collection.fetch().done(function(){
+                if(self.content_rating_collection.models.length == 1) {
+                    self.rating_view.model = self.content_rating_collection.pop();
+                    self.rating_view.render();
+                } else {
```

Comment:
This else statement handles the case of having more than 1 content rating model. It will never happen in the backend, but you never know. Can you print out an error saying "Something's wrong, you shouldn't have more than 1 content rating model!"